app/repository/family.py

```diff
@@ -116,14 +116,22 @@ def _update_intention(
     return update_title, update_basics, update_metadata, update_collections
 
 
-def all(db: Session) -> list[FamilyReadDTO]:
+def all(db: Session, org_id: int, is_superuser: bool) -> list[FamilyReadDTO]:
```

Comment:
Just a thought ... I'm wondering if we could pass a `UserContext` object around at some point with these (and other) attribute in?

Reply:
Yeah I like that idea, a lot less janky!

Reply:
https://linear.app/climate-policy-radar/issue/PDCT-1113/pass-usercontext-object-around-instead-of-user-properties-individually